### DISCUSSION TITLE
Website still broken

### DISCUSSION THREAD

**Discussion by johnlamox** [elrumo/macOS_Big_Sur_icons_replacements#1343] [Help with the website]
Hi there,

does anybody know when the website will be fixed finally? All of my icons are still broken. 25 still waiting for approval.
This really sucks! :-(

**Comment by elrumo** [comment]
Hey! The website should be working fine, what problem do you have?

**Reply by Circa-III** [reply]
I think they are mostly referring to new icons not being approved. I also have a few still pending.

**Reply by johnlamox** [reply]
No! All of my icons are still broken and can't be downloaded.
![Image2](https://user-images.githubusercontent.com/57749200/149518738-f4c0da8d-301f-48dc-8036-9abc74071668.png)
When I click on one icon, i get a blank page. When i download it with right-clicking, the file is corrupt.
I hope I don't have to upload all Icons again.

**Reply by ThaliaMyers** [reply]
I ended up having to upload all of my icons again for them to be available because it wasn't being fixed.

**Reply by johnlamox** [reply]
Now they completely disappeared. Totally sucks! :-(

**Comment by shiro6x** [comment]
Resources page don't work

**Reply by elrumo** [reply]
I'm away this week, but I'll get it fixed for good next week.

**Reply by elrumo** [reply]
I finally came around fixing it last week!
